src/graphics/vulkan/vulkan_buffer.cpp

```
@@ -27,7 +29,7 @@ namespace canyon::graphics::vulkan {
     }
 
     void* Buffer::Map() {
```

Clang (14.0.6)

[error] method 'Map' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)